tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -50,6 +52,19 @@ public class ImageTests
                 new ImageDescriptor { RuntimeVersion = "4.8", BuildVersion = "4.8", OsVariant = WSC_1903 },
             };
 
+        private static ImageDescriptor[] WCFTestData = new ImageDescriptor[]
```

Comment:
Normal coding convention for acronyms is to only capitalize the first letter: `WcfTestData`

Reply:
will do in the next commit